documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,120 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Attribute Index Recovery
+
+
+The below outlined steps refer to recovery of Attribute Index Table Segments but only in cases of Metadata Segments which are a special case, as they need to
```

Comment:
Suggested rephrase: `This document refers to recovery of Attribute Index Table Segments but...`

Reply:
corrected.